### PR TITLE
Morning commit: Review submission finished, filtering partially done

### PR DESCRIPTION
**Pull request by markus-leben**

### CODE REVIEW

File: client/src/index.jsx
[the remark this answers is not in the document]
Comment (Jenessap) [reply client/src/index.jsx:1]: Actually, never mind my previous comment. I came up with a better fix. I created a temporary branch called mergeMnM (for "merge Markus and Main"), resolved the merge conflicts that were there, and submitted a pull request to your branch from mergeMnM. Would you take a look at the ways I resolved those conflicts, and if they're acceptable, merge them with your branch, and push them? Then those resolutions will be reflected here, right?  After we see the resolutions as part of your code here, I'll merge this branch with main, ok?

File: example.env
Comment (Jenessap) [inline example.env:1]: Thank you for adding this.

**Review by Jenessap — CHANGES_REQUESTED** [review]
I made a temporary merge branch called mergeMnM to resolve merge conflicts, and then I made a pull request to your branch from there. Could you take a look at those and update this PR with those changes if the resolutions I made there are acceptable to you? Thanks.
Also, on line 89 of our shared index.jsx file (top level) there's this: "refresh={refreshProducts}". When I go to render the page, I get the error that refreshProducts is undefined. Would you define it here or address it however you'd like?

File: client/src/index.jsx
[the remark this answers is not in the document]
Comment (markus-leben) [reply client/src/index.jsx:1]: Your temp branch dropped necessary changes to support my component so I just manually resolved the conflicts.

**Pull request merged**